### DISCUSSION TITLE
Is there a way to change active tab styles?

### DISCUSSION THREAD

**Discussion by didemkkaslan** [tamagui/tamagui#2548] [Q&A]
I've used the default tabs example to make this. Is there a way I can change the active tab styles?

<img width="369" alt="Screenshot 2024-04-17 at 08 19 32" src="https://github.com/tamagui/tamagui/assets/106841409/c192664d-bdbc-4183-aa37-b78c56c947bd">

**Comment by davidgg** [comment]
This is the best way I've found, probably there are other better ways.

Style the tab:

```ts
const StyledTab = styled(Tabs.Tab, {
  variants: {
    active: {
      true: {
        backgroundColor: "#3498db",
      },
    },
  },
});
```

And use it, passing the active value:
```tsx
 <Tabs.List
        separator={<Separator vertical />}
        disablePassBorderRadius="bottom"
        aria-label="Manage your account"
      >
        <StyledTab flex={1} value="tab1" active={activeTab === "tab1"}>
          <SizableText fontFamily="$body">Profile</SizableText>
        </StyledTab>
```
